src/contrib/dependencyinjection/Akka.DI.Core/Akka.DI.Core.csproj

```diff
@@ -3,7 +3,7 @@
 
   <PropertyGroup>
     <AssemblyTitle>Akka.DI.Core</AssemblyTitle>
-    <Description>Dependency injection support for Akka.NET</Description>
+    <Description>Dependency injection support for Akka.NET [DEPRECATED - replaced by Akka.DependencyInjection]</Description>
```

Comment:
Deprecation warning in the description for the old package. I'll also mark the package as deprecated on NuGet and in a subsequent PR, mark the old code as obsolete.